config/initializers/secret_token.rb

```diff
@@ -1,10 +1,12 @@
-require 'securerandom'
-
-token = $redis.get("SECRET_KEY_BASE")
-if token.nil?
-  token = SecureRandom.hex(64)
-  $redis.set("SECRET_KEY_BASE", token)
-end
-
-Rails.application.config.secret_token = token
-Rails.application.config.secret_key_base = token
+# require 'securerandom'
```

Comment:
What is the reason for commenting this?

Reply:
Yeah, about that. `config/application.rb` created a new instance of `Redis` (with default values). But for Docker we need to use `REDIS_URL`, which is not yet available in `application.rb`. 
Also, _using_ Redis in initializers requires you have redis setup when running _any_ rake task (like `assets:precompile` and `db:migrate`). 

I'm not sure what the use-case was for storing the secret token in Redis, I assume keeping it alive between deployments. I've now defined it in `.env`/`env.example` instead. 

If the above is okay with you, the whole `config/initializers/secret_token.rb` can be deleted.

Reply:
@ariejan It was something I took over from GitLab. Mostly so users don't have to generate their own key. That was something that went wrong quite some times over there. The secret key needs to be a certain length for it to work, and people tend to mess that up. So for me that was the reason to generate it on the fly.

How do you think we can tackle the "human" issue?

Reply:
Ah yeah! That sounds good.

Reply:
I've update setup to ask for all info that's needed to generate a `.env`. `SECRET_KEY_BASE` is generated as well.